docs/src/content/docs/snippets/grant-create.mdx

```diff
@@ -14,7 +14,7 @@ Before a client can call the Open Payments APIs, it must request and receive a g
 These code snippets enable a client to request a grant for the `incoming-payment`, `quote`, and `outgoing-payment` resource types. Each request includes all valid actions the client can take for the resource type.
 
 :::note
-When setting up a payment, the client first requests a grant from the recipient's (payee's) authorization server to create an `incoming-payment` resource. The client then requests grants to create the `quote` and `outgoing-payment` resources from the sender's (payer's) authorization server.
+When setting up a payment, the client first requests a grant from the authorization server of the recipient's <Tooltip content="Account servicing entity" client: load><span>ASE</span></Tooltip> to create an `incoming-payment` resource. The client then requests grants to create the `quote` and `outgoing-payment` resources from the authorization server of the sender's ASE.
```

Comment:
Given that the sentence above the note specifies incoming-payment, quote, and outgoing-payment resource types, what do you think about this:

When setting up a payment, the client first requests an incoming payment grant from the authorization server of the recipient's ASE. Then, the client requests quote and outgoing payment grants from the authorization server of the sender's ASE.

Just a suggestion.